latex/expressingParallelism.tex

```diff
@@ -2249,6 +2249,66 @@ \subsubsection{Defining kernels using OpenCL C kernel objects}
 \codeinline{cl_kernel} is no longer needed in the calling context.
 
 
+%*******************************************************************************
+% OpenCL Kernel Conventions and SYCL
+%*******************************************************************************
+\subsection{OpenCL Kernel Conventions and SYCL}
+\label{sec:opencl_kern_conventions_and_sycl}
+
+OpenCL and SYCL use opposite conventions for the unit stride dimension.  SYCL
```

Comment:
I think we should outline here what the C++ and OpenCL conventions are to set the foundation for the rest of this section.

Reply:
I also think we should clarify that the OpenCL convention here is not madated by OpenCL, but is inferred and most commonly adopted by implementations.

Reply:
I tried to outline the convention and clarify.  Please let me know if you think it should be improved in any way.

Reply:
This addition looks good, that covers what I had in mind, thanks.